src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -358,6 +358,7 @@ public Description getDescription() {
     public void run(final RunNotifier notifier) {
         EachTestNotifier testNotifier = new EachTestNotifier(notifier,
                 getDescription());
+        testNotifier.fireTestSuiteStarted();
```

Comment:
I am a developer of [Allure Framework](http://allure.qatools.ru/). We are using `RunListener` to collect information about tests. This change can help me to get more information about test suites - as example get the right start/stop times for it.  

BTW the problem with parameterized runner above -is not a problem for me, but I would like to choose the right way